mathics/doc/common_doc.py

```diff
@@ -1058,6 +1072,9 @@ def __init__(self, module=None):
                 and var.__module__[: len(self.pymathicsmodule.__name__)]
                 == self.pymathicsmodule.__name__
             ):  # nopep8
+                if not check_requires_list(var):
+                    print("  skiping ", var.__name__)
```

Comment:
skiping -> skipping. But I'd be more explicit to why it is skipped, an optional package is not installed.

Reply:
This print actually never happens, because we are not generating documentation for pymathics modules. I just removed it.